explore/src/main/scala/explore/EditableLabel.scala

```diff
@@ -59,29 +69,72 @@ object EditableLabel {
       addButtonLabel,
       addButtonClass,
       editButtonClass,
+      editButtonTooltip,
       deleteButtonClass,
-      deleteButtonIcon
+      deleteButtonIcon,
+      deleteButtonTooltip,
+      okButtonTooltip,
+      discardButtonTooltip
     )
 
+  type Editing = Editing.Type
```

Comment:
Can this just be an `enum Editing: case NotEditing, InEdition` ?

Reply:
Wouldn't be cheaper to use an opaque type?

Reply:
Generally speaking, yes. If we define `val`s for the values, probably yes too, but no so much ;) Ohh unless the vals are inline?